services/provider/server/server.go

```diff
@@ -664,6 +672,14 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 					"clusterID": getSubVolumeGroupClusterID(subVolumeGroup),
 					"csi.storage.k8s.io/snapshotter-secret-name": provisionerSecretName,
 				})})
+
+			extR = append(extR, &pb.ExternalResource{
+				Name: "cephfs",
+				Kind: "VolumeGroupSnapshotClass",
+				Data: mustMarshal(map[string]string{
+					"clusterID": getSubVolumeGroupClusterID(subVolumeGroup),
+					"csi.storage.k8s.io/snapshotter-secret-name": provisionerSecretName,
```

Comment:
Each append has the potential to grow the capacity of the baking storage of the slice. That means coping of the entire slice data. 

I would suggest refactoring the code so we will have a single append operation:
```
extR = append(
    extR, 
    &pb.ExternalResource{
    },
    &pb.ExternalResource{
    },
    &pb.ExternalResource{
    },
)
```